Dockerfile-kpromo

```diff
@@ -19,13 +19,13 @@ ARG OS_CODENAME
 ARG DISTROLESS_IMAGE
 FROM golang:${GO_VERSION}-${OS_CODENAME} as builder
 
-WORKDIR /go/src/k8s.io/release
+WORKDIR /go/src/sigs.k8s.io/k8s-container-image-promoter
 
 # Copy the sources
 ENV package="./cmd/kpromo"
-COPY ../../go.mod ../../go.sum ./
-COPY ../../pkg ./pkg/
-COPY ../../cmd/kpromo ${package}/
+
+# TODO(kpromo): Consider restricting the files copied in after refactor
```

Comment:
Since these are going to a builder image and we don't have large blobs in the repo perhaps the impact of fine-tuning the COPYs is negligible?